conf/messages

```diff
@@ -68,20 +68,22 @@ home.unique.taxpayer.reference                                  = Unique Taxpaye
 home.utr                                                        = UTR: {0}
 home.payments.heading                                           = Next payments due
 home.payments.no-payments-due                                   = No payments due
-home.payments.view                                              = View what you owe
+home.payments.view                                              = Check what you owe
 home.updates.heading                                            = Next updates due
-home.updates.view                                               = View next updates
+home.updates.view                                               = View update deadlines
+home.paymentHistoryAndCredit.heading                            = Payment history and credit
+home.paymentHistoryAndCredit.view                               = Payment and refund history
 home.overdue.date.payment.count                                 = {0} OVERDUE PAYMENTS
 home.overdue.date.update.count                                  = {0} OVERDUE UPDATES
 home.overdue.date                                               = OVERDUE
-home.tax-years.heading                                          = Tax years
-home.tax-years.view                                             = View tax years
+home.tax-years.heading                                          = Returns
+home.tax-years.view                                             = View all tax years
 home.tax-years.viewPayments                                     = View payment history
```

Comment:
@andriikondiuk - I don't think this is being used anymore.

Reply:
Done, thanks